dotCMS/src/integration-test/java/com/dotcms/security/apps/AppsAPIImplTest.java

```diff
@@ -794,6 +796,42 @@ public void Test_Get_Secret_fallbackOnSystemHost_With_Non_Valid_License()
         nonValidLicenseAppsAPI.getSecrets("anyKey", true, systemHost, admin);
     }
 
+    /**
+     * Given scenario: We simulate a non valid license situation then we call  AppsAPI#exportSecrets
+     * Expected Results: we should get an InvalidLicenseException
+     * @throws DotDataException
+     * @throws DotSecurityException
+     */
+    @Test(expected = InvalidLicenseException.class)
+    public void Test_Export_With_Non_Valid_License()
+            throws DotDataException, DotSecurityException, IOException {
+
+        final User admin = TestUserUtils.getAdminUser();
+        //AES only supports key sizes of 16, 24 or 32 bytes.
+        final String password = RandomStringUtils.randomAlphanumeric(32);
+        final Key key = AppsUtil.generateKey(password);
+        nonValidLicenseAppsAPI.exportSecrets(key,true,ImmutableMap.of(),admin);
```

Comment:
Codacy found an issue: [Unnecessary use of fully qualified name 'ImmutableMap.of' due to existing static import 'com.google.common.collect.ImmutableMap.of'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6286472)